### ISSUE TITLE
Accessibility and internals

### ISSUE THREAD

**Issue by Gouvernathor** [Gouvernathor/Politiscales-Angular#2]
For accessibility reasons, every changeable field should be an Angular Form field.
Since Reactive Forms are annoying, when Signal Forms are available (Nov 2025 and Angular 21 should be), the page should be converted to that system.
